src/plugins/cloudinary/common.js

```diff
@@ -50,7 +51,7 @@ const isSrcEqual = (source1, source2) => {
 
 export const extendCloudinaryConfig = (currentConfig, newConfig) => Object.assign(currentConfig, newConfig);
 
-export const getCloudinaryUrl = (publicId, transformation) => createCloudinaryLegacyURL(publicId, transformation);
+export const getCloudinaryUrl = (publicId, transformation) => createCloudinaryLegacyURL(publicId, omit(transformation, ['chainTarget']));
```

Comment:
Let's figure out what's `chainTarget` and if it's even needed

Reply:
its need for chaining , so you can do:
player.transformation().source